### PR TITLE
Review rules

### PR DESCRIPTION
**Pull request by meadows115**

### CODE REVIEW

File: rules/codelingo/effective-go/comment-first-word-when-empty/codelingo.yaml [hunk 1]
```diff
@@ -21,6 +21,22 @@ funcs:
...
+          The first word in the summary should be the function name, followed by a description. 
+          
+          ## Bad code
+          ```
+          func Unnamed(){}
+          ```
+          ## Good code
+          ```
+          //Unnmaded is the first word and this is a summary of the function. 
+          func Unnamed(){}
```
Comment (epben) [inline rules/codelingo/effective-go/comment-first-word-when-empty/codelingo.yaml]: Small typo here and a single space is normal between the `//` and the comment.

I'd also change the examples to use something more realistic, and then layout the function like:
```
func Decrypt(data []byte) {
   ...
}
```

File: rules/codelingo/effective-go/comment-first-word-when-empty/codelingo.yaml [hunk 2]
```diff
@@ -21,6 +21,22 @@ funcs:
...
+          ```
       codelingo/review:
         comment: |
           Every exported function in a program should have a doc comment. The first sentence should be a summary that starts with the name ({{funcName}}) being declared.
```
Comment (epben) [inline rules/codelingo/effective-go/comment-first-word-when-empty/codelingo.yaml]: Perhaps wrap ({{funcName}}) in backticks to get markdown code formatting? i.e.

`({{funcName}})`

File: rules/codelingo/effective-go/comment-first-word-when-empty/codelingo.yaml [hunk 3]
```diff
@@ -21,6 +21,22 @@ funcs:
...
+          to allow for an automated presentation of these doc comments.
+          The first word in the summary should be the function name, followed by a description. 
+          
+          ## Bad code
+          ```
+          func Unnamed(){}
+          ```
+          ## Good code
+          ```
+          //Unnmaded is the first word and this is a summary of the function.
```
Comment (Daanikus) [inline rules/codelingo/effective-go/comment-first-word-when-empty/codelingo.yaml]:
```suggestion
          // Unnamed is the first word and this is a summary of the function. 
```

File: rules/codelingo/effective-go/comment-first-word-when-empty/codelingo.yaml [hunk 4]
```diff
@@ -21,6 +21,22 @@ funcs:
 tenets:
   - name: comment-first-word-when-empty
     actions:
+      codelingo/docs:
+        title: Comment First Word as Subject
```
Comment (Daanikus) [inline rules/codelingo/effective-go/comment-first-word-when-empty/codelingo.yaml]:
```suggestion
        title: Comment First Word When Empty
```

**Review by Daanikus — CHANGES_REQUESTED** [review]

**Review by Daanikus — APPROVED** [review]

**Pull request merged**